pkg/ui/src/views/jobs/jobTable.tsx

```diff
@@ -84,7 +84,16 @@ export class JobTable extends React.Component<JobTableProps, JobTableState> {
   }
 
   componentDidUpdate(prevProps: Readonly<JobTableProps>): void {
-    this.setCurrentPageToOneIfJobsChanged(prevProps);
+    if (prevProps.jobs !== this.props.jobs) {
```

Comment:
why is this diff here?

Reply:
Independent merge skew blocking master, but this diff doesn't actually help. Removed to fix only the windows thing, though now I won't be able to merge it until @dhartunian fixes the other problem.

Reply:
I think you can merge it by rebasing your branch off the merge commit where this bug was introduced.